Dockerfile

```diff
@@ -20,6 +20,7 @@ ENV SLAVE_DESCRIPTION="Core Jenkins Slave"
 
 # Pre-requisites
 RUN yum -y install epel-release
+RUN yum -y install dos2unix
```

Comment:
You should avoid makin a new layer (you added new line with RUN). Instead you should add "dos2uinix" next to epel-release, so like multiple packages are executed in one command.